karamel-core/src/main/java/se/kth/karamel/cookbook/metadata/MetadataRb.java

```diff
@@ -18,76 +20,48 @@
   String name;
   String description;
   String version;
-  String resultsDir;
-  String buildCommand;
-  String urlBinary;
-  String urlGitclone;
   List<Recipe> recipes = new ArrayList<>();
   List<Attribute> attributes = new ArrayList<>();
-
+  
   public void setAttributes(List<Attribute> attributes) {
     this.attributes = attributes;
   }
-
+  
   public List<Attribute> getAttributes() {
     return attributes;
   }
-
-  public String getUrlBinary() {
-    return urlBinary;
```

Comment:
I left these in to discuss whether it would be useful to have them or not. But fine, we should remove them.